cves/CVE-2018-12460.yml

```diff
@@ -129,7 +132,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The lifetime for this vulenrability is short and sweet. With the addition 
+    of the fix for the vulnerability occurring only two months after the 
+    vulenrability's creation. No major refactoring of the file took place during
+    that time.  One developer added support code that led to the vulenrability
```

Comment:
vulnerability is spelled wrong on lines 136, 138, 139 and 140

Reply:
fixed spelling and grammar in paragraph